### PR TITLE
[2778] Update ChatTheme docs

### PR DESCRIPTION
**Pull request by MarinTolic**
### 🎯 Goal

Update ChatTheme docs with all of the relevant new parameters that have been added throughout the development process.

### 🧪 Testing

Please run Docusaurus to make sure everything renders correctly.

#### General
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-team or #compose-chat-sdk-team) (required)
- [x] PR targets the `develop` branch

#### Code & documentation
- [x] Changelog is updated with client-facing changes
- [ ] ~~New code is covered by unit tests~~
- [ ] ~~Comparison screenshots added for visual changes~~
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

docs

### CODE REVIEW

**Review by bychkovdmitry — APPROVED** [review]
Very good!

File: docusaurus/docs/Android/04-compose/02-general-customization/01-chat-theme.mdx [hunk 1]
```diff
@@ -92,30 +94,85 @@ Let's see what each property exposes and what the values are used for.
...
 `StreamShapes` provides a small collection of shapes that let us style our containers.
 
-You can find all the shapes we expose in [the class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamShapes.kt), as well as what the default shapes are.
+You can find all the shapes we expose in the [class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamShapes.kt), as well as what the default shapes are.
 
 These are really easy to customize, as you've seen before, and can make your app feel closer to your design system.
+
+### StreamAttachmentFactories
+
+`StreamAttachmentFactories` provides default factories for uploads, links, both Giphy and regular images and files.
```
Comment (skydoves) [inline docusaurus/docs/Android/04-compose/02-general-customization/01-chat-theme.mdx:127]:
```suggestion
`StreamAttachmentFactories.defaultFactories()` provides default factories for uploads, links, both Giphy and regular images and files.
```

File: docusaurus/docs/Android/04-compose/02-general-customization/01-chat-theme.mdx [hunk 2]
```diff
@@ -92,30 +94,85 @@ Let's see what each property exposes and what the values are used for.
...
+
+`AttachmentPreviewHandler.defaultAttachmentHandlers()` provides default handlers for media, document and url attachments
+
+If you want you can take a look at the [class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/filepreview/AttachmentPreviewHandler.kt).
+
+You can customize file previews by creating your own list of `AttachmentPreviewHandler`s and overriding `ChatTheme.attachmentPreviewHandlers` with it.
+
+### DefaultReactionTypes
+
+Used for defining reactions the user can add to messages. Provides five basic reactions out of the box.
```
Comment (skydoves) [inline docusaurus/docs/Android/04-compose/02-general-customization/01-chat-theme.mdx:141]:
```suggestion
Used for defining reactions the user can add to messages. `DefaultReactionTypes.defaultReactionTypes()` provides five basic reactions out of the box.
```

**Review by skydoves — APPROVED** [review]
Looks great to me! 👍

File: docusaurus/docs/Android/04-compose/02-general-customization/01-chat-theme.mdx [hunk 3]
```diff
@@ -92,30 +94,85 @@ Let's see what each property exposes and what the values are used for.
...
+
+### AttachmentPreviewHandler
+
+`AttachmentPreviewHandler.defaultAttachmentHandlers()` provides default handlers for media, document and url attachments
+
+If you want you can take a look at the [class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/filepreview/AttachmentPreviewHandler.kt).
+
+You can customize file previews by creating your own list of `AttachmentPreviewHandler`s and overriding `ChatTheme.attachmentPreviewHandlers` with it.
+
+### DefaultReactionTypes
```
[the remark this answers is not in the document]
Comment (filbabic) [reply docusaurus/docs/Android/04-compose/02-general-customization/01-chat-theme.mdx:139]: This is definitely something we need to improve in the future, to allow custom emoji in the form of `Painter`s. I'll open a ticket for the refactoring of reactions.

**Review by filbabic — APPROVED** [review]
Looks good! Minor changes, added docs comments.

Nicely done! :]

**Pull request merged**